app/views/layouts/admin/_page.html.slim

```diff
@@ -1,6 +1,13 @@
 - content_for :styles
   = stylesheet_link_tag "admin/content", :media => "all"
 
+- content_for :scripts
+
+  javascript:
```

Comment:
Please put this javascript block at the bottom of the template. This is our general convention (I know there are exceptions in this project).